modules/storages/spec/features/storages/admin/create_storage_spec.rb

```diff
@@ -50,10 +50,9 @@
 
       expect(page).to be_axe_clean.within "#content"
 
-      within(".PageHeader-titleBar") do
-        click_on("Storage")
-        within_test_selector("storages-select-provider-action-menu") { click_on("Nextcloud") }
-      end
+      page.find_test_selector("storages-create-new-provider-button").click
```

Comment:
🔴 I'd rather keep the `within` and `click_on("Storage")`. Just the selector changes to `within(".SubHeader")`. It is intentional, that those tests are written with real UI text, to test those pages more axe_clean.

Reply:
Fixed 👍